src/hubspot-client.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.6;net46</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
**comment**: moving to netstandard2.0 is actually a breaking change. I don't mind it, we should move forward.

Reply:
Right, forgot to mention that. I do believe it's good to stay up to date with Nugets to prevent "bit rot" in your own library :)

.NET 5.0 is even released already, it's moving real fast 😳

Reply:
totally agree, and yes, things are moving really fast.